packages/core/src/App/Containers/Layout/header/traders-hub-header.jsx

```diff
@@ -45,78 +48,31 @@ export const TradersHubHomeButton = ({ is_dark_mode }) => {
     );
 };
 
-const TradingHubOnboarding = ({ is_dark_mode, setIsOnboardingVisited }) => {
-    const history = useHistory();
-    return (
-        <div className='trading-hub-header__tradinghub--onboarding'>
-            <div className='trading-hub-header__tradinghub--onboarding--logo'>
-                <Popover
-                    classNameBubble='account-settings-toggle__tooltip'
-                    alignment='bottom'
-                    message={<Localize i18n_default_text='View onboarding' />}
-                    should_disable_pointer_events
-                    zIndex={9999}
-                >
-                    <Icon
-                        icon={is_dark_mode ? 'IcAppstoreTradingHubOnboardingDark' : 'IcAppstoreTradingHubOnboarding'}
-                        size={20}
-                        onClick={() => {
-                            history.push(routes.onboarding);
-                            setIsOnboardingVisited(false);
-                        }}
-                    />
-                </Popover>
-            </div>
-        </div>
-    );
-};
-
-const ShowNotifications = ({ is_notifications_visible, notifications_count, toggleNotifications }) => {
-    return (
-        <div className='trading-hub-header__notification'>
-            <ToggleNotifications
-                count={notifications_count}
-                is_visible={is_notifications_visible}
-                toggleDialog={toggleNotifications}
-                tooltip_message={<Localize i18n_default_text='View notifications' />}
-            />
-        </div>
-    );
-};
-
-const TradingHubHeader = ({
+const TradersHubHeader = ({
+    account_type,
+    balance,
+    country_standpoint,
+    currency,
+    has_any_real_account,
     header_extension,
+    is_acc_switcher_disabled,
+    is_acc_switcher_on,
```

Comment:
what's the difference between disabled and on?

Reply:
god knows 🚨